app/views/payload_application_processes/show.html.erb

```diff
@@ -23,7 +23,7 @@
             </div>
             <div class="group bg-zinc-800 dark:bg-white/2.5">
               <div class="relative">
-                <%= render 'json_output', transformed_data: transformed_data(job_application) %>
+                <%= render 'json_output', job_application:, questions: job_application.application_criterion.questions %>
```

Comment:
The ApplicationCriterion is slightly bothersome - is there a way we can get around this?

Reply:
We could setup custom inflection rules to handle this

Reply:
I am not sure to follow you here ...